src/widgets/torrent/TorrentQueueItem.tsx

```diff
@@ -18,17 +41,29 @@ export const BitTorrrentQueueItem = ({ torrent }: TorrentQueueItemProps) => {
   return (
     <tr key={torrent.id}>
       <td>
-        <Tooltip position="top" withinPortal label={torrent.name}>
-          <Text
-            style={{
-              maxWidth: '30vw',
-            }}
-            size="xs"
-            lineClamp={1}
-          >
-            {torrent.name}
-          </Text>
-        </Tooltip>
+        <Popover opened={popoverOpened} width={350} position="top" withinPortal>
```

Comment:
I don't think there is a need to set the position. having it not set allows it to automatically place as it should

Reply:
Note: I reverted your removal of the width here, because this is not wanted behavior.
It is unusable when making the widget fairly small.